go.mod

```diff
@@ -20,4 +20,4 @@ require (
 
 replace github.com/metal3-io/baremetal-operator/apis => ./apis
 
-replace github.com/metal3-io/baremetal-operator/ironic => ./pkg/ironic
+replace github.com/metal3-io/baremetal-operator/pkg/ironic => ./pkg/ironic
```

Comment:
Is it really necessary any more?

Reply:
@dtantsur yep, if we change something in ./pkg/ironic inside local change, without rewrite rule gomod will try to fetch from remote location, and code would be without newer change

Reply:
we should keep that rewrite

Reply:
Yes, to be compatible with `replace github.com/metal3-io/baremetal-operator/apis => ./apis` usage. I'm not sure, but if we don't do that local changes in ironic package can not be seen in main module.